src/components/OSCALMarkupProse.js

```diff
@@ -1,32 +1,35 @@
 import React from "react";
 import ReactMarkdown from "react-markdown";
 
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+
+const components = {
+  a: ({ ...props }) =>
+    props.href.startsWith("http") ? (
+      <a target="_blank" {...props}>
+        <OpenInNewIcon /> {props.children}
+      </a>
+    ) : (
+      <a {...props}>{props.children}</a>
+    ),
+};
```

Comment:
I'd like to name this something a little more descriptive like `baseComponents` or something like that.